### PR TITLE
Feature: show watched & ratings

### PR DESCRIPTION
**Pull request by katjam**
It's messy but it works. You have to nuke your `data.db` on first run so the extra fields make it in.

Feel free to refactor.

### CODE REVIEW

File: update_watchlist.py [hunk 1]
```diff
@@ -100,24 +137,41 @@ def process_watchlist():
...
+def extract_watched():
+    zip_file = os.path.join(MY_PATH, ZIP_FILE)
+    with zipfile.ZipFile(zip_file, 'r') as zip_ref:
+        zip_ref.extract(WATCHED_CSV, path=os.path.join(MY_PATH, DATA))
+
+def extract_ratings():
+    zip_file = os.path.join(MY_PATH, ZIP_FILE)
+    with zipfile.ZipFile(zip_file, 'r') as zip_ref:
+        zip_ref.extract(RATINGS_CSV, path=os.path.join(MY_PATH, DATA))
+    os.remove(zip_file)
```
Comment (katjam) [inline update_watchlist.py:174]: Remove here since this is the last extract we run. Could be better.

File: show_watchable.py [hunk 2]
```diff
@@ -67,6 +67,13 @@ def run():
             print(stylize(title, GREEN))
         print()
 
+def watched_text():
+    return "Seen it!"
+
+def rated_text(rating):
+    popcorns = "\U0001F600" * rating
```
Comment (katjam) [inline show_watchable.py:74]: sadly, had to go with smileys.

We could put in a CONST

File: show_watchable.py [hunk 3]
```diff
@@ -67,6 +67,13 @@ def run():
             print(stylize(title, GREEN))
         print()
 
+def watched_text():
+    return "Seen it!"
```
Comment (katjam) [inline show_watchable.py:71]: This could probably be inlined. I was going to build a bit more complicated.

**Review by bananaoomarang — APPROVED** [review]

**Comment by katjam** [comment]
Thanks! Hopefully you will enjoy the new feature.

**Pull request merged**